python/ray/tests/test_modin.py

```diff
@@ -0,0 +1,520 @@
+# Licensed to Modin Development Team under one or more contributor license
+# agreements. See the NOTICE file distributed with this work for additional
+# information regarding copyright ownership.  The Modin Development Team
+# licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+# This file is copied and adapted from:
+# http://github.com/modin-project/modin/master/modin/pandas/test/test_general.py
+# http://github.com/modin-project/modin/master/modin/pandas/test/utils.py
+
+import pandas
+import pytest
+import modin
+import modin.pandas as pd
+import numpy as np
+from numpy.testing import assert_array_equal
+from modin.utils import to_pandas
+from pandas.testing import (assert_series_equal, assert_frame_equal,
+                            assert_extension_array_equal, assert_index_equal)
+from packaging import version
+import ray
+from ray.util.client.ray_client_helpers import ray_start_client_server
+
+# Versions of modin prior to 0.9.1 are incompatible with ray
+# client. See https://github.com/modin-project/modin/pull/2851
+modin_version = version.parse(modin.__version__)
+pytestmark = pytest.mark.skipif(
+    modin_version <= version.parse("0.9.1"),
+    reason="Incompatible modin version")
+
```

Comment:
but we are using the nightly was right? so this is not needed

Reply:
True, I'll drop that